src/AccessibilityInsights.SharedUx/Settings/ConfigurationConstants.cs

```diff
@@ -71,6 +69,7 @@ public partial class ConfigurationModel
         private const string keyTestReportPath = "TestReportPath";
         private const string keyTreeViewMode = "TreeViewMode";
         private const string keyVersion = "Version";
-        private const string keyZoomLevel = "ZoomLevel";
+        private const string keyIssueReporting= "IssueReportingGuid";
+        private const string keyIssueReporterSerializedConfig = "IssueReporterSerializedConfig";
```

Comment:
Does this mean that if I want to use both GitHub and ADO, I need to log in every time I switch? I'd sure like to be able to persist within each provider...

Reply:
No, data will be saved for all extensions simultaneously. You're in luck.